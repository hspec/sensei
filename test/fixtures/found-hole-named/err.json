{"version":"1.0","ghcVersion":"ghc-9.10.0","span":{"file":"test/fixtures/found-hole-named/Foo.hs","start":{"line":3,"column":5},"end":{"line":3,"column":9}},"severity":"Error","code":88464,"message":["Found hole: _foo :: A\nOr perhaps `_foo' is mis-spelled, or not in scope","In an equation for `a': a = _foo","Relevant bindings include\n  a :: A (bound at test/fixtures/found-hole-named/Foo.hs:3:1)\nValid hole fits include\n  a :: A (bound at test/fixtures/found-hole-named/Foo.hs:3:1)"],"hints":[]}
