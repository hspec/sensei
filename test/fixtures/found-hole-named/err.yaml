version: '1.0'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/found-hole-named/Foo.hs
  start:
    line: 3
    column: 5
  end:
    line: 3
    column: 9
severity: Error
code: 88464
message:
- |-
  Found hole: _foo :: A
  Or perhaps `_foo' is mis-spelled, or not in scope
- 'In an equation for `a'': a = _foo'
- |-
  Relevant bindings include
    a :: A (bound at test/fixtures/found-hole-named/Foo.hs:3:1)
  Valid hole fits include
    a :: A (bound at test/fixtures/found-hole-named/Foo.hs:3:1)
hints: []
