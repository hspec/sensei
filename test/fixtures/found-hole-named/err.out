test/fixtures/found-hole-named/Foo.hs:3:5: error: [GHC-88464]
    • Found hole: _foo :: A
      Or perhaps `_foo' is mis-spelled, or not in scope
    • In an equation for `a': a = _foo
    • Relevant bindings include
        a :: A (bound at test/fixtures/found-hole-named/Foo.hs:3:1)
      Valid hole fits include
        a :: A (bound at test/fixtures/found-hole-named/Foo.hs:3:1)

